Tasks/Common/webdeployment-common/kuduutility.ts

```diff
@@ -86,7 +48,10 @@ export async function deployWebAppPackage(webAppPackage: string, publishingProfi
     };
     if(takeAppOfflineFlag) {
         tl.debug('Trying to enable app offline mode.');
-        await appOffineKuduService(publishingProfile.publishUrl, physicalPath, headers, true); 
+        var appOfflineFilePath = path.join(tl.getVariable('system.DefaultWorkingDirectory'), 'app_offline_temp.htm');
+        tl.writeFile(appOfflineFilePath, '<h1> Site under construction. </h1>');
```

Comment:
Earlier it used to be "The Web Page is temporarily unavailable". Is this change per PMs?